### ISSUE TITLE
Please upload weight file

### ISSUE THREAD

**Issue by whdgusdl48** [kravrolens/RECNet#3]
Hello,

I conducted the experiment myself because the weight file was not available, but I couldn't achieve the performance mentioned in the paper. Could you please upload the weight file?

The last recorded PSNR in my experiments did not exceed 23.

Thank you.

**Comment by CastoHu** [comment]
> Hello,
> 
> I conducted the experiment myself because the weight file was not available, but I couldn't achieve the performance mentioned in the paper. Could you please upload the weight file?
> 
> The last recorded PSNR in my experiments did not exceed 23.
> 
> Thank you.

Hello, I would like to know what dataset configuration you used for training? I used the MSEC dataset mixed with the LCDP dataset, but the validation set results show that the PSNR of the model can reach 26-28 only under single exposure conditions, and the PSNR of the LCDP validation set in this mixed dataset training is only about 13.

**Comment by zyjakira666** [comment]
> Hello,
> 
> I conducted the experiment myself because the weight file was not available, but I couldn't achieve the performance mentioned in the paper. Could you please upload the weight file?
> 
> The last recorded PSNR in my experiments did not exceed 23.
> 
> Thank you.

which dataset you train is 23
